packages/server/src/schema/metadata/resolvers.ts

```diff
@@ -34,56 +36,546 @@ export const resolvers: Resolvers = {
-        },
-      }
     },
     version() {
       return '2.0.0'
     },
   },
 }
+
+function getIri(id: number | string): string {
+  return `https://serlo.org/${id}`
+}
+
+function getSchemaType(entityType: string): string {
+  switch (entityType) {
+    case 'article':
+      return 'Article'
+    case 'course':
+      return 'Course'
+    case 'text-exercise-group':
+    case 'text-exercise':
+      return 'Quiz'
+    case 'video':
+      return 'VideoObject'
+    case 'applet':
+      return 'WebApplication'
+    default:
+      return entityType
+  }
+}
+
+function getLearningResourceType(entityType: string): { id: string }[] {
+  return getOpeneduhubIds(entityType)
+    .map((vocab) => `http://w3id.org/openeduhub/vocabs/new_lrt/${vocab}`)
+    .map((id) => ({ id }))
+}
+
+function getOpeneduhubIds(entityType: string): string[] {
+  switch (entityType) {
+    case 'article':
+      return ['588efe4f-976f-48eb-84aa-8bcb45679f85']
+    case 'course':
+      return ['ad9b9299-0913-40fb-8ad3-50c5fd367b6a']
+    case 'text-exercise':
+    case 'text-exercise-group':
+      return ['a33ef73d-9210-4305-97f9-7357bbf43486']
+    case 'video':
+      return ['7a6e9608-2554-4981-95dc-47ab9ba924de']
+    case 'applet':
+      return ['4665caac-99d7-4da3-b9fb-498d8ece034f']
+    default:
+      return []
+  }
+}
+
+function getSerloOrganizationMetadata() {
+  return {
+    id: 'https://serlo.org/organization',
+    type: 'Organization',
+    name: 'Serlo Education e.V.',
+  }
+}
+
+function toSubject(data: RawSubject): Subject {
+  return {
+    type: 'Concept',
+    id: toIdString(data.scheme) + data.id,
+    inScheme: { id: toSchemeString(data.scheme) },
+  }
+}
+
+enum Scheme {
+  UniversitySubject,
+  SchoolSubject,
+}
+
+function getRaWSubject(id: number): RawSubject[] {
+  switch (id) {
+    // Mathematik (Schule)
+    case 5:
+    case 23593:
+    case 141587:
+    case 169580:
+      return [{ id: '1017', scheme: Scheme.SchoolSubject }]
+
+    // Nachhaltigkeit => Biologie, Ethik (Schule)
+    case 17744:
+    case 48416:
+    case 242851:
+      return [
+        { id: '1001', scheme: Scheme.SchoolSubject },
+        { id: '1008', scheme: Scheme.SchoolSubject },
+      ]
+    // Chemie (Schule)
+    case 18230:
+      return [{ id: '1002', scheme: Scheme.SchoolSubject }]
+    // Biologie (Schule)
+    case 23362:
+      return [{ id: '1001', scheme: Scheme.SchoolSubject }]
+    // Englisch (Shule)
+    case 25979:
+    case 107557:
+    case 113127:
+      return [{ id: '1007', scheme: Scheme.SchoolSubject }]
+    // Latain (Schule)
+    case 33894:
+    case 106085:
+      return [{ id: '1016', scheme: Scheme.SchoolSubject }]
+    // Physik (Schule)
+    case 41107:
+      return [{ id: '1022', scheme: Scheme.SchoolSubject }]
+    // Informatik (Schule)
+    case 47899:
+      return [{ id: '1013', scheme: Scheme.SchoolSubject }]
+
+    // Politik => Politik, Sachunterricht (Schule)
+    case 79159:
```

Comment:
```suggestion
    case 79159:
    case 107556:
```
or did you change it on purpose compared to https://github.com/serlo/database-layer/blob/0574bf5805bf8bed63006c06d574f9a572e9ece5/server/src/metadata.rs#L553C21-L553C27 ?